Misc/NEWS.d/next/Library/2022-01-27-13-30-02.bpo-46544.oFDVWj.rst

```diff
@@ -0,0 +1,2 @@
+Don't leak ``x`` & ``uspace`` intermediate vars in
+:class:`textwrap.TextWrapper`.
```

Comment:
A news entry for such internal changes seems to have low value for end users!

Reply:
It's useful in the unlikely case someone is relying on these attributes.